model/change_point_detection.go

```diff
@@ -369,10 +369,21 @@ func ReplaceChangePoints(ctx context.Context, env cedar.Environment, performance
 	return catcher.Resolve()
 }
 
-func GetTotalPagesForChangePointsGroupedByVersion(ctx context.Context, env cedar.Environment, projectId string, pageSize int) (int, error) {
-	countKey := "count"
-	pipe := appendAfterBaseGetChangePointsByVersionAgg(projectId, bson.M{
-		"$count": countKey,
+type GetChangePoinsGroupedByVersionArgs struct {
```

Comment:
This is misspelled, also I would change it to `ChangePointsByVersionOpts`. (We call these arg structs opts instead of args).